meldingen_core/actions/melding.py

```diff
@@ -97,6 +99,30 @@ async def __call__(self, pk: int, values: dict[str, Any], token: str) -> T:
         return melding
 
 
+class MeldingAddContactAction(BaseCRUDAction[T, T_co]):
+    """Action that adds contact information to a melding."""
+
+    _verify_token: TokenVerifier[T, T_co]
+
+    def __init__(
+        self,
+        repository: BaseMeldingRepository[T, T_co],
+        token_verifier: TokenVerifier[T, T_co],
+    ) -> None:
+        super().__init__(repository)
+        self._verify_token = token_verifier
+
+    async def __call__(self, pk: int, phone: str, email: str, token: str) -> T:
```

Comment:
The `phone` and `email` arguments should be optional I think. Could you change that?